### PR TITLE
Run OLM and OperatorHub tests as a separate suite in CI

### PR DESCRIPTION
**Pull request by spadgett**
@alecmerdler We're planning on switching the OperatorHub and OLM tests to run as a separate suite in CI. These tests will still be enabled by default for merge.

cc @jwforres @enj

### CODE REVIEW

**Comment by spadgett** [comment]
This will require an additional PR to openshift/release to add the new status (but this PR needs to merge first)

**Comment by spadgett** [comment]
Corresponding release PR: https://github.com/openshift/release/pull/3088

File: frontend/integration-tests/protractor.conf.ts [hunk 1]
```diff
@@ -91,11 +91,14 @@ export const config: Config = {
     newApp: ['tests/base.scenario.ts', 'tests/overview/overview.scenario.ts', 'tests/source-to-image.scenario.ts', 'tests/deploy-image.scenario.ts'],
     olm: ['tests/base.scenario.ts', 'tests/olm/descriptors.scenario.ts', 'tests/olm/catalog.scenario.ts', 'tests/olm/prometheus.scenario.ts', 'tests/olm/etcd.scenario.ts'],
     olmUpgrade: ['tests/base.scenario.ts', 'tests/olm/update-channel-approval.scenario.ts'],
+    operatorHub: ['tests/base.scenario.ts', 'tests/operator-hub/operator-hub.scenario.ts'],
+    // OLM and OperatorHub
+    olmFull: ['tests/base.scenario.ts', 'tests/operator-hub/operator-hub.scenario.ts', 'tests/olm/descriptors.scenario.ts', 'tests/olm/catalog.scenario.ts', 'tests/olm/prometheus.scenario.ts', 'tests/olm/etcd.scenario.ts'],
```
Comment (alecmerdler) [inline frontend/integration-tests/protractor.conf.ts]: We don't need this suite; the OLM and Operator Hub suites should always be ran separately.

**Comment by spadgett** [comment]
cc @jeff-phillips-18

File: frontend/integration-tests/protractor.conf.ts (hunk 1 above)
Comment (spadgett) [reply frontend/integration-tests/protractor.conf.ts]: The reason to group them would be to avoid a separate install for each suite when run as part of CI

**Pull request merged**